src/pages/en/guides/server-side-rendering.mdx

```diff
@@ -117,6 +117,20 @@ export const prerender = true;
 </html>
 ```
 
+[MDX pages](/en/guides/markdown-content/#using-exported-variables-in-mdx) can also be prerendered:
```

Comment:
I like @Jutanium's suggestion to remove "any" pages (since clearly, that's not true). I think if we do that, then BOTH  the `.astro` and the `.mdx` examples need an introductory line, because now MDX is not an "also" thing.

Maybe something like:

> Add `export const prereder = true` to any `.astro` or `.mdx` page in the appropriate place to render it statically:

And then show the two code snippets? Something like that? What do we think?

Reply:
Yeah, just saying "pages" implies that _all_ pages can be prerendered, which is not the case. The Pages docs say:

> Astro supports the following file types in the src/pages/ directory:
> 
> [.astro](https://docs.astro.build/en/core-concepts/astro-pages/#astro-pages)
> [.md](https://docs.astro.build/en/core-concepts/astro-pages/#markdownmdx-pages)
> .mdx (with the [MDX Integration installed](https://docs.astro.build/en/guides/integrations-guide/mdx/#installation))
> [.html](https://docs.astro.build/en/core-concepts/astro-pages/#html-pages)
> .js/.ts (as [endpoints](https://docs.astro.build/en/core-concepts/endpoints/))

So I think it makes sense to be explicit that only `.astro`, `.mdx`, and endpoints can be prerendered. I'll add an example for `.js/.ts`.